libiio.iss.cmakein

```diff
@@ -44,11 +44,11 @@ Name: "turkish"; MessagesFile: "compiler:Languages\Turkish.isl"
 Name: "ukrainian"; MessagesFile: "compiler:Languages\Ukrainian.isl"
 
 [Files]
-Source: "C:\projects\libiio\build-win32\Release\libiio.dll"; DestDir: "{sys}"; Flags: 32bit
-Source: "C:\projects\libiio\build-win64\Release\libiio.dll"; DestDir: "{sys}"; Check: Is64BitInstallMode
+Source: "C:\projects\libiio\build-win32\Release\libiio.dll"; DestDir: "{sys}"; Flags: 32bit ignoreversion
+Source: "C:\projects\libiio\build-win64\Release\libiio.dll"; DestDir: "{sys}"; Check: Is64BitInstallMode; Flags: ignoreversion
 
-Source: "C:\projects\libiio\build-win32\tests\Release\*.exe"; DestDir: "{sys}"; Check: not Is64BitInstallMode
-Source: "C:\projects\libiio\build-win64\tests\Release\*.exe"; DestDir: "{sys}"; Check: Is64BitInstallMode
+Source: "C:\projects\libiio\build-win32\tests\Release\*.exe"; DestDir: "{sys}"; Check: not Is64BitInstallMode; Flags: ignoreversion
+Source: "C:\projects\libiio\build-win64\tests\Release\*.exe"; DestDir: "{sys}"; Check: Is64BitInstallMode; Flags: ignoreversion
 
 Source: "C:\projects\libiio\build-win32\Release\libiio.lib"; DestDir: "{pf32}\Microsoft Visual Studio 12.0\VC\lib"; Check: not Is64BitInstallMode
```

Comment:
Should these get the 'ignoreversion' flag as well?

Reply:
If they don't have the flag, they will not be copied in the system. This means that we could install a new libiio, use the latest library version, but not that latest iio_info for example.

Reply:
Sorry, got this question wrong. I checked the logs and, unlike .exe and.dll, the .lib file does not have a file version at all. This means that the installer will not take into consideration any version, will check the timestamps and copy the provided file.

This is a snapshot from the log file:
` Dest filename: C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\lib\amd64\libiio.lib`
` Time stamp of our file: 2020-05-28 07:02:02.000`
`  Dest file exists.`
` Time stamp of existing file: 2019-01-10 15:19:26.000`
` Version of our file: (none)`
` Version of existing file: (none)`
` Installing the file.`